keyboards/crkbd/keymaps/foostan/keymap.c

```diff
@@ -0,0 +1,242 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
```

Comment:
The `QMK_KEYBOARD_H` include includes `bootloader.h` automatically.

```suggestion
```

Reply:
Thanks. fixed 🙏